Modules/Quotas/test/Quotas.Domain.Tests/Tests/Identities/IdentityTests.cs

```diff
@@ -79,16 +80,16 @@ public void Can_delete_tier_quota_by_definition_id_with_multiple_quotas()
     }
 
     [Fact]
-    public void Cannot_delete_nonexistent_tier_quota_by_definition_id()
+    public void Trying_to_delete_inexistent_quota_throws_NotFoundException()
```

Comment:
I mixed something up here.
```suggestion
    public void Trying_to_delete_inexistent_quota_throws_DomainException()
```